src/blockchain/Blockchain.ts

```diff
@@ -272,6 +273,20 @@ export class Blockchain {
         return this.networkConfig
     }
 
+    /**
+     * @returns Current PrevBlocksInfo
+     */
+    get prevBlocks(): PrevBlocksInfo | undefined {
+        return this.prevBlocksInfo
+    }
+
+    /**
+     * Sets PrevBlocksInfo.
+     * @param value PrevBlocksInfo to set
+     */
+    set prevBlocks(value: PrevBlocksInfo | undefined) {
```

Comment:
Maybe support value as Cell? So the users could pass it directly without preparsing steps